### PR TITLE
feat: override docker and helm package metadata 

### PR DESCRIPTION
**Pull request by gberche-orange**
Relates to https://github.com/renovatebot/renovate/issues/18436#issuecomment-1288014919

## Documentation (please check one with an [x])

- [ ] I have updated the documentation, or
- [x] No documentation update is required

## How I've tested my work (please select one)

I have verified these changes via:

- [ ] Code inspection only, or
- [ ] Newly added/modified unit tests, or
- [ ] No unit tests but ran on a real repository, or
- [ ] Both unit tests + ran on a real repository

### CODE REVIEW

File: lib/modules/datasource/metadata-manual.ts [hunk 1]
```diff
@@ -46,14 +46,27 @@ export const manualChangelogUrls: Record<string, Record<string, string>> = {
     wagtail: 'https://github.com/wagtail/wagtail/tree/master/docs/releases',
   },
   docker: {
+    'confluentinc/ksqldb-server': 'https://github.com/confluentinc/ksql',
+    'confluentinc/ksqldb-cli': 'https://github.com/confluentinc/ksql',
+    'crossplanecontrib/provider-helm': 'https://github.com/crossplane-contrib/provider-helm',
+    'crossplanecontrib/provider-kubernetes': 'https://github.com/crossplane-contrib/provider-kubernetes',
+    'prom/blackbox-exporter': 'https://github.com/prometheus/blackbox_exporter',
```
Comment (rarkins) [inline lib/modules/datasource/metadata-manual.ts:53]: Can this be reordered alphabetically? :)

Comment (gberche-orange) [reply lib/modules/datasource/metadata-manual.ts]: done with force push, sorry I had missed this one.

File: lib/modules/datasource/metadata-manual.ts [hunk 2]
```diff
@@ -46,14 +46,27 @@ export const manualChangelogUrls: Record<string, Record<string, string>> = {
     wagtail: 'https://github.com/wagtail/wagtail/tree/master/docs/releases',
   },
   docker: {
+    'confluentinc/ksqldb-cli': 'https://github.com/confluentinc/ksql',
```
Comment (viceice) [inline lib/modules/datasource/metadata-manual.ts:49]: those all should be in manuals source urls below instead.

**Review by viceice — CHANGES_REQUESTED** [review]

File: lib/modules/datasource/metadata-manual.ts (hunk 2 above)
Comment (gberche-orange) [reply lib/modules/datasource/metadata-manual.ts]: thanks @viceice, resolved in  a073acf

**Review by viceice — DISMISSED** [review]

**Review by rarkins — APPROVED** [review]

**Pull request merged**